types/src/chainspec/vm_config/system_config.rs

```diff
@@ -112,11 +112,14 @@ impl SystemConfig {
         // this causes flaky test failures if the random result exceeds i64::MAX
         let install_upgrade_gas_limit = rng.gen::<u32>() as u64;
         let standard_transaction_gas_limit = rng.gen::<u32>() as u64;
-        let auction_costs = rng.gen();
+        let mut auction_costs: AuctionCosts = rng.gen();
         let mint_costs = rng.gen();
         let handle_payment_costs = rng.gen();
         let standard_payment_costs = rng.gen();
 
+        // workaround for the test flakiness issue mentioned above
+        auction_costs.change_bid_public_key = rng.gen::<u32>() as u64;
+
```

Comment:
you should be able to add this field to the Standard distribution impl of Auction Costs, avoiding the need to do a mut twiddle here.